### ISSUE TITLE
Missing/Deprecated library

### ISSUE THREAD

**Issue by Seraf** [beego/bee#446]
Hello,

I was trying to upgrade my bee library through govendor, and I got an error when launching bee : 
`vendor/github.com/beego/bee/cmd/commands/dlv/dlv_amd64.go:34:2: cannot find package "github.com/derekparker/delve/terminal" in any of:`

It seems that github.com/derekparker/delve/terminal doesn't exist anymore.
Could be worth to remove every occurences of bee utility pointing to this ?

Thanks

**Comment by astaxie** [comment]
But the dependence is already in vendor? I have no idea on that

**Comment by amrfaissal** [comment]
@Seraf Seems they moved stuff around again. Can you skip delve while upgrading your vendor folder for the time being?

**Comment by Seraf** [comment]
Yes for sure, I will upgrade only bee, it was just to tell you to change it on future versions, because not everyone is using govendor and have a copy of the old files ;-)

**Comment by sinkcup** [comment]
@astaxie this error is because repo had been renamed, so you should change depends.

old: github.com/derekparker/delve
new: github.com/go-delve/delve
